ortho.py

```diff
@@ -0,0 +1,68 @@
+from enum import Enum, auto
+
+
+class Position(Enum):
+    ORIGIN = auto()
+    HOP = auto()
+    OTHER = auto()
+
+
+def create(a, b, c, d):
```

Comment:
Is there a longer name that serves as a good description of `a`, `b`, `c`, and `d` here? If not, you might consider `x1`, `x2`, `x3`, and `x4` or `x11`, `x12`, `x21`, `x22`?